src/public/index.html

```diff
@@ -7,10 +7,34 @@
     <meta name="description" content="A blog about something awesome!">
     <title>My Blog</title>
   </head>
+  <header>
```

Comment:
Everything that is not metadata about the HTML page must appear inside the `<body>` tags. This `<header>` tag will not be visible and will technically be invalid HTML.